js/content-slider.js

```diff
@@ -1,85 +1,81 @@
 export default class ContentSlider {
   /**
-   * TODO: 
-   * !! - Fix the Controls repositioning logic in the resizeObserver. 
+   * TODO:
+   * !! - Fix the Controls repositioning logic in the resizeObserver.
    * !! - Investigate the reason the controls click area diverges from img div on different browser zoom levels
    *  - Fix tab indexes (user should not be able to scroll by tabbing // add visibility classes )
-   *  - Refactor addition of event listeners into a separate function. DRY-er. 
-   * 
+   *  - Refactor addition of event listeners into a separate function. DRY-er.
+   *
    * Notes:
    *  - Clickable area off by 20px due to 2rem padding on .demo-section
    *  - Viewport top position start at [0, 79], not sure why.
    */
-  constructor(sliderWrapper = '.content-slider') {
-    // check if there is a content slider
-        this.contentSlider = document.querySelector(sliderWrapper);
+  constructor(
+    sliderWrapper = ".content-slider",
+    controlSurface = ".content-block__upper"
```

Comment:
Seeing this along with the other PR, I wonder about just giving ourselves an easy way to distill `controlSurface` based on the wrapper, rather than needing to pass in a selector for every custom item. Like, maybe putting a `data-slider-controlsurface="content-block__upper"` or something like that on the wrapper.